designer/app.py

```diff
@@ -1015,6 +1016,7 @@ def action_btn_save_pressed(self, exit_on_save=False, *args):
         else:
             self.save_project()
             if exit_on_save:
+                self.remove_temp_proj_directories()
```

Comment:
Ok, originally I wasn't sure whether I should mess with _perform_quit due to the "_" prefix, that should simplify the code. I'll work on it later today.